### ISSUE TITLE
Is it compatible with python 3?

### ISSUE THREAD

**Issue by geoffreysandja** [syrusakbary/pyjade#159]
we are trying to move our whole projects to python 3, i would like to know if it's compatible and safe.

**Comment by ekaradon** [comment]
I do not think it is. I just tried something really simple and it did not work as expected. I am maybe doing something wrong, but I think that there may some compatibility issues. 
See, when doing something like: 

```jade
		if debug
			span DEBUG == TRUE
		else
			SPAN DEBUG == FALSE
```

I am getting always a false result while settings.py set debug to True. 
So I am inclined to think that you should not be using pyjade with python 3. 
Furthermore, it is written as not compatible here: 
https://www.djangopackages.com/packages/p/pyjade/

**Comment by chompomonim** [comment]
+1 for python3

**Comment by gsekando** [comment]
+1

**Comment by Xender** [comment]
+1

**Comment by ptrdvds** [comment]
@ekaradon In python(3) TRUE and FALSE are not boolean, True and False are.
Have been using pyjade with jinja2, python3 and flask andd it works perfectly.

**Comment by tdamsma** [comment]
For me Jade + Python3 + Pyramid is also working fine
